packages/hiro-graph-client/src/client.js

```diff
@@ -566,7 +566,7 @@ export default class Client {
     /**
      *  Write timeseries values (only ogit/Timeseries vertices)
      *
-     *  values are { timeseriesId: string, value: {timestamp: number; value: PlainObject | Array<PlainObject>}}
+     *  values are { timestamp: millisecond unix, value: PlainObject | number | string | Array<string | number | PlainObject[]> }
```

Comment:
Needs to be changed here as well, please.

Reply:
done! Thanks a lot! Type hell